samples/crypto/pbkdf2/README.rst

```diff
@@ -0,0 +1,52 @@
+.. _crypto_pbkdf2:
+
+Crypto: PBKDF2
+##############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Password Based Key Derivation Function (PBKDF2) sample shows how to derive keys with the PBKDF2 algorithm, using a sample password salt, and iteration count.
+The underlying pseudorandom function (PRF) used in this sample is HMAC with SHA-256.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+Overview
+********
+
+The sample performs the following operations:
+
+1. Initialization of the Platform Security Architecture (PSA) API.
+
+#. Key derivation:
+
+   a. The input key is imported into the PSA crypto keystore.
+   #. The output key is derived.
+
+#. Cleanup:
+
+   a. The input and the output keys are removed from the PSA crypto keystore.
```

Comment:
I've updated to say input password, instead of input key, since we are deriving from a password in this sample.

Also, the sample is using output_bytes, instead of output_key, so there is no output key after the key derivation.
I'm removing any mention of output key in the key store.

The end results is the derived key, in the application, instead of the key store.
Usually you would want to keep the derived key in the key_store and only use it by reference, but in this case we are outputting the key to the application